src/mescal/tapis_v3.clj

```diff
@@ -243,4 +280,4 @@
   [base-url token-info-fn timeout storage-system irods-path]
   (when-not (string/blank? irods-path)
     (let [root-dir (get-root-dir base-url token-info-fn timeout storage-system)]
-      (string/replace irods-path (re-pattern (str "\\Q" root-dir)) ""))))
+      (string/replace irods-path (re-pattern (str "^\\Q" root-dir)) ""))))
```

Comment:
During testing in QA, a path like `/cyverse/home/psarando/analyses/cyverse-qa-test` would become `/home/psarando/analyses/qa-test` without this fix.